### DISCUSSION TITLE
OpenGL (Legacy) flickering during gameplay

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#22963] [Q&A]
Looking at the video here, cursor and UI text seem to dissapear for a frame. Setting the Frame Limiter to Vsync alleviates the issue. This issue does not affect UI on the other game screens.
This issue started happening on version 2023.326.0

https://user-images.githubusercontent.com/53346335/233840321-9c9284e3-6b26-4512-a0b3-500ebe954198.mp4

Logs:
[runtime.log](https://github.com/ppy/osu/files/11072320/runtime.log)

**Comment by tropicaaal** [comment]
Having similar trouble - although i've had issues with this on every renderer, and it's also affecting *some* UI elements. VSync fixes it, but it's not ideal since VSync seems to cause noticable mouse latency for me (no idea why).

Video demonstrating the flickering on Direct3D 11

https://user-images.githubusercontent.com/42101043/227813753-77c47d34-8a09-4ccc-bc76-08788203094d.mp4

**Reply by Theighlin** [reply]
Did it happen before the new release? If so, it's likely a driver issue (make sure they're updated). If it didn't, upload logs, might be rare software hardware combination.

**Reply by tropicaaal** [reply]
Ah, didn't even realize they were out of date. Seems to not be an issue after updating graphics drivers.

**Comment by Sinamer** [comment]
Opened an issue instead: https://github.com/ppy/osu/issues/23294
